skins/laika/tests/unit/components/shared/BankData.spec.ts

```diff
@@ -215,6 +216,52 @@ describe( 'BankData', () => {
 		expect( iban.text() ).toMatch( 'Test Bank' );
 	} );
 
+	it( 'renders bank code / BIC field when legacy bank account number is entered', async () => {
+		const wrapper = mount( BankData, {
+			localVue,
+			store: createStore(),
+			mocks: {
+				$t: () => {},
+			},
+		} );
+
+		let bankInfoInput = wrapper.find( 'input#bic' );
+		expect( bankInfoInput.isVisible() ).toBe( false );
+
+		wrapper.setData( { accountId: '123' } );
+		await wrapper.vm.$nextTick();
+
+		bankInfoInput = wrapper.find( 'input#bic' );
+		expect( bankInfoInput.isVisible() ).toBe( true );
+	} );
+
+	it( 'renders bank code / BIC field when valid IBAN was entered and validated', async () => {
+		const wrapper = mount( BankData, {
+			localVue,
+			store: createStore(),
+			mocks: {
+				$t: () => {},
+			},
+		} );
+		const iban = wrapper.find( '#iban' );
+		const store = wrapper.vm.$store;
+		store.dispatch = jest.fn();
+
+		let bankInfoInput = wrapper.find( 'input#bic' );
+		expect( bankInfoInput.isVisible() ).toBe( false );
+
+		wrapper.setData( { accountId: 'DE89370400440532013000' } );
+
+		// this should be done differently...
+		await wrapper.vm.$nextTick();
+		await store.dispatch( NS_BANKDATA, setBankData );
+		await wrapper.vm.$nextTick();
+		// ...
```

Comment:
do you have an idea how to mock/await the validation of the iban here? @gbirke @Abban 
the desired result should be the store getter `bankdataIsValid` to be `true`. Then the bank field input should be visible.
 (alternatively: how to mock a computed value?)

Reply:
If you fil the #iban field with the desired data then trigger a blur you should be able to then see if the bic field is visible on the page or not.